clients/netcore/DataAPIClient.gen.cs

```diff
@@ -4868,6 +5245,52 @@ public partial class UInt64ReadCreateResponseModel
 
     }
 
+    [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "13.15.10.0 (NJsonSchema v10.6.10.0 (Newtonsoft.Json v13.0.0.0))")]
+    public partial class UpdateProofLocationModel
+    {
+        /// <summary>
+        /// Uri to retrieve the ProofLocation from the Trust API.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty("uri", Required = Newtonsoft.Json.Required.Always)]
+        [System.ComponentModel.DataAnnotations.Required(AllowEmptyStrings = true)]
+        public string Uri { get; set; }
+
+        [Newtonsoft.Json.JsonProperty("validationResult", Required = Newtonsoft.Json.Required.Always)]
+        public ValidateProofResultEnumeration ValidationResult { get; set; }
+
+        /// <summary>
+        /// Timestamp of the latest Validation
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty("validationTimestamp", Required = Newtonsoft.Json.Required.Always)]
+        [System.ComponentModel.DataAnnotations.Required(AllowEmptyStrings = true)]
+        public System.DateTimeOffset ValidationTimestamp { get; set; }
+
+    }
+
+    [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "13.15.10.0 (NJsonSchema v10.6.10.0 (Newtonsoft.Json v13.0.0.0))")]
+    public enum ValidateProofResultEnumeration
```

Comment:
is this enum customer facing? if so please make sure the enum is generated with tags to give the entries proper names